pype/plugins/global/publish/integrate_new.py

```diff
@@ -81,7 +81,8 @@ class IntegrateAssetNew(pyblish.api.InstancePlugin):
                 "assembly",
                 "fbx",
                 "textures",
-                "action"
+                "action",
+                "template"
```

Comment:
I think we should be more specific with this family. Template sounds too generic, considering it's only harmony compatible family. How about `harmony.template` or something specific like that

Reply:
Sure, `harmony.template` sounds good.
Was looking at `nukenodes` but like the dot notation better.

Reply:
Done